### ISSUE TITLE
Add definite vector-valued example, compressible elasticity

### ISSUE THREAD

**Issue by jedbrown** [jedbrown/dohp#3]
Useful for testing performance and correctness for vector-valued problems without the complexity of mixed spaces and indefinite preconditioning.  Do we have exact solutions for nonlinear elasticity or will we need to manufacture them?

**Comment by jedbrown** [comment]
commit fd8b403d51bd805b324cfd6e64dbcc1bc3c4971a
Author: Jed Brown jed@59A2.org
Date:   Sun May 10 00:20:53 2009 +0200

```
Add elasticity example

* The shell Jacobian and preconditioner (for linear elements) is correct
  even with the nonlinear strain tensor.

* Spectral convergence to manufactured solution is observed for a single element.

* There is some error with high order and multiple elements.  This needs
  to be fixed before implementing an indefinite vector problem (Stokes).

References gh-3
```

**Comment by jedbrown** [comment]
Add elasticity test.  This is a nonlinear vector problem and it is large
enough that it takes a couple seconds to run.  It uses the (now default)
RCM ordering.

Closed by 806d0f971baa207efbe3aa4cbe2eca91d3c9062f.
